tests/acceptance/uninstall/UninstallExtensionCest.php

```diff
@@ -28,6 +28,7 @@ public function uninstallExtension(AcceptanceTester $I, $scenario)
 		$I->doAdministratorLogin();
 		$I->amOnPage('/administrator/index.php?option=com_installer&view=manage');
 		$I->click("//button[@class='btn hasTooltip js-stools-btn-filter']");
+		$I->wait(1);
```

Comment:
`$I->waitForElementVisible('#filter_type', 30);`

Reply:
yeah @puneet0191  . it will be the best option .